src/MessageCenter/MessageCenter.jsx

```diff
@@ -0,0 +1,151 @@
natos (Greek: Σπυρίδων Μαρινάτος; 17 November [O.S. 4 November] 1901[a] – 1 October 1974) was a Greek archaeologist who specialised in the Minoan and Mycenaean civilizations of the Aegean Bronze Age. He is best known for the excavation of the Minoan site of Akrotiri on Thera,[b] which he conducted between 1967 and 1974. He received many honours in Greece and abroad, and was considered one of the most important Greek archaeologists of his day.  A native of Kephallonia, Marinatos was educated at the University of Athens, the Friedrich Wilhelms University of Berlin, and the University of Halle. His early teachers included noted archaeologists such as Panagiotis Kavvadias, Christos Tsountas and Georg Karo. He joined the Greek Archaeological Service in 1919, and spent much of his early career on the island of Crete, where he excavated several Minoan sites, served as director of the Heraklion Museum, and formulated his theory that the collapse of Neopalatial Minoan society had been the result of the eruption of the volcanic island of Thera around 1600 BCE.  In the 1940s and 1950s, Marinatos surveyed and excavated widely in the region of Messenia in south-west Greece, collaborating with Carl Blegen, who was engaged in the simultaneous excavation of the Palace of Nestor at Pylos. He also discovered the battlefield of Thermopylae and excavated the Mycenaean cemeteries at Tsepi and Vranas near Marathon in Attica.  Marinatos was head of the Greek Archaeological Service from 1937 to 1939, from 1955 to 1958, and and lastly from 1967 to 1974, under the Greek military junta. He was an enthusiastic supporter of the junta; in the late 1930s, he had been close to the quasi-fascist dictatorship of Ioannis Metaxas, under whom he initiated legislation to restrict the roles of women in Greek archaeology. His leadership of the Archaeological Service has been criticised for its cronyism and for promoting the pursuit of grand discoveries at the expense of good scholarship. Marinatos died while excavating at Akrotiri in 1974, and is buried at the site.  Life Early career and education Spyridon Marinatos was born in Lixouri on the Ionian island of Kephallonia on 17 November [O.S. 4 November] 1901. His father, Nikolaos, was a carpenter.[2] Marinatos studied at the University of Athens from 1916,[3] where he competed unsuccessfully with Christos Karouzos for a scholarship, beginning a lifelong rivalry between the two.[4] Marinatos joined the Greek Archaeological Service in 1919 and was first posted to Crete as an epimelitis (junior archaeological official).[5] His early excavations on Crete included the Minoan villa at Amnisos,[6] and he continued to excavate on the island periodically between 1919 and 1952.[7]  Marinatos was one of the first thirty-six students of the "Practical School of Art History", an archaeological training centre established by the Archaeological Society of Athens at the request of the Greek government, studying there in the 1919–1920 academic year.',
```

Comment:
You can remove this commented variable since you are pulling the data from the backend now.